src/web/app/pages-logs/logs-page.component.html

```diff
@@ -135,10 +135,19 @@ <h1>Logs</h1>
 </div>
 
 <div *ngIf="isTableView">
+  <div class="center mb-4" *ngIf="hasResult && !hasPreviousPage">
+    <button class="btn btn-light" (click)="extendStartTime()">Extend start time by 10 minutes</button>
+  </div>
+  <div *tmIsLoading="isSearching" class="center mb-4">
+    <button *ngIf="hasResult" id="load-previous-button" class="btn btn-primary" (click)="loadPreviousLogs()" [disabled]="!hasPreviousPage">Load more</button>
```

Comment:
Instead of disabling when there is no previous page, can just remove from view entirely. It's made obsolete by the extension button anyway.

Reply:
Updated